src/system/applications/item/injury-sheet.ts

```diff
@@ -50,11 +51,25 @@ export class InjuryItemSheet extends BaseItemSheet {
     public async _prepareContext(
         options: DeepPartial<foundry.applications.api.ApplicationV2.RenderOptions>,
     ) {
+        if (
+            this.item.system.description!.value ===
+            CONFIG.COSMERE.items.types.injury.desc_placeholder
+        ) {
+            this.item.system.description!.value = game.i18n!.localize(
+                this.item.system.description!.value!,
+            );
+        }
+
+        const enrichedDescValue = await TextEditor.enrichHTML(
+            this.item.system.description!.value!,
+        );
+
         return {
             ...(await super._prepareContext(options)),
             isPermanent:
                 this.item.system.type === InjuryType.PermanentInjury ||
                 this.item.system.type === InjuryType.Death,
+            descHtml: enrichedDescValue,
```

Comment:
Can't we move the description enrichment into the base item?

Reply:
Done the refactor to make the description code common. I don't think we can move the localisation of the values into the data models due to the way the mixin process works, the i18n objects isn't populated on the game object when the mixins are initialised (if I'm following your thought correctly).
I did try and provide the ancestry DescriptionItemMixin call the localised string when I was first implementing the initial values bit as it seemed more sense to me to just do it cleanly there, but alas it didn't work.